code/obj/critter/limbed_critter.dm

```diff
@@ -0,0 +1,70 @@
+ABSTRACT_TYPE(/obj/critter/limbed_critter)
+/obj/critter/limbed_critter
```

Comment:
not a huge fan of this, this would be better done with components

Reply:
Agreed